README.rst

```diff
@@ -60,12 +62,27 @@ Activate the add-on by installing it in the Addons Control panel.
 Configure
 ---------
 
-To create a vocabulary from an index, go to the 'Index Vocabularies' control
-panel and select the indexes you wish to create vocabularies for.
 
-To use the vocabulary in a tag field, you will need to edit the XML Schema of
-your content type. Currently this is only possible via the Classic UI
-(Control Panel -> Content Types -> Schema -> Edit XML Schema)
+To create a custom tag field on a content type you will need to do the following
+
+1. Create a new field of type "Multiselect"
+2. Save the content type. This will create an index for this field.
+3. Go to Dynamic Vocabularies in the control panel
+4. Select the Index matching the field you just created and click "Make Vocabulary".
+5. Copy the name of the vocabulary generated.
+8. Edit your content type again but this time via the editing it's XML
+   - Currently this is only possible via the Classic UI (Control Panel -> Content Types -> Schema -> Edit XML Schema)
+9. Change the widget to a TagSelectWidget with the vocabulrary name that was generated.  
+
+ e.g. 
```

Comment:
```suggestion
   .. code-block:: xml
```

Reply:
You might have to fiddle with indentation to get the XML to render with proper syntax highlighting. In general, I always indent my enumerated lists and code-blocks with 4 spaces. Your enumerated lists are indented with only 3 spaces, which can cause funky rendering in reStructuredText.